math/subset_sum/info.toml

```
@@ -0,0 +1,23 @@
+title = 'Subset Sum'

```

```suggestion
title = '$#_p$ Subset Sum'
```

元のissueにもありましたが、subset sumだと判定版っぽいので(https://en.wikipedia.org/wiki/Subset_sum_problem)